### PR TITLE
Austin dustin

### PR DESCRIPTION
**Pull request by NicholasPierce1**

### CODE REVIEW

**Comment by NicholasPierce1** [comment]
folder:
repository

File: src/main/java/com/webapp/TextBook/repository/Bag/BagRepository.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.webapp.TextBook.repository.Bag;
+
```
Comment (NicholasPierce1) [inline src/main/java/com/webapp/TextBook/repository/Bag/BagRepository.java:2]: blah

File: src/main/java/com/webapp/TextBook/repository/Bag/BagRepository.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import com.webapp.TextBook.repository.BookCopy.BookCopyRepositoryCustom;
+import com.webapp.TextBook.repository.data_access.Bag;
+import com.webapp.TextBook.repository.data_access.BookCopy;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface BagRepository extends JpaRepository<Bag, UUID>, BagRepositoryCustom {
```
Comment (NicholasPierce1) [inline src/main/java/com/webapp/TextBook/repository/Bag/BagRepository.java:12]: typo please revise

File: src/main/java/com/webapp/TextBook/controller/web_controller/HomeController.java [hunk 3]
```diff
@@ -1,42 +1,118 @@
...
+
+                return "login";
+            }
+        }
+        catch (Exception e){
+            //Catching Exceptions and printing StackTrace of the error
+            //Loading in custom status message and returning to login page
+            System.out.println(e.getStackTrace());
+            map.addAttribute("Error", null);
+            map.addAttribute("StatusMessage", "Internal error has occurred. " +
```
Comment (Infraction71) [inline src/main/java/com/webapp/TextBook/controller/web_controller/HomeController.java:98]: The FitnessGram Pacer Test is a multistage aerobic capacity test that progressively gets more difficult as it continues. The 20 meter pacer test will begin in 30 seconds. Line up at the start. The running speed starts slowly but gets faster each minute after you hear this signal bodeboop. A sing lap should be completed every time you hear this sound. ding Remember to run in a straight line and run as long as possible. The second time you fail to complete a lap before the sound, your test is over. The test will begin on the word start. On your mark. Get ready!… Start.

Comment (NicholasPierce1) [reply src/main/java/com/webapp/TextBook/controller/web_controller/HomeController.java:98]: https://github.com/NicholasPierce1/TextBookServices/commit/fefa1bcf3c2b9de1242bee44ff1911d469e5c366

**Review by Infraction71 — APPROVED** [review]
Looks good

**Review by Infraction71 — APPROVED** [review]

**Pull request closed without merging**